apps/jobs/views.py

```diff
@@ -372,6 +372,19 @@ def change_submission_data_and_visibility(
         if is_public is True:
             when_made_public = datetime.datetime.now()
             request.data["when_made_public"] = when_made_public
+
+        submissions_already_public = Submission.objects.filter(
```

Comment:
Hey @Ram81 Why are we using filter here? I think get should work here because a participant can only one submission public and if they change to another then the previous submission should be made private which is one in number.